vivisect/__init__.py

```diff
@@ -2758,9 +2760,20 @@ def loadFromFile(self, filename, fmtname=None, baseaddr=None):
             return self.normFileName(filename)
 
         mod = viv_parsers.getParserModule(fmtname)
+
+        # get baseaddr and size, then make sure we have a good baseaddr
+        baseaddr, size = mod.getMemBaseAndSize(self, filename=filename, baseaddr=baseaddr)
+        logger.debug('initial baseva: 0x%x  size: 0x%x', baseaddr, size)
+        if baseaddr == 0:
```

Comment:
i'm open to any suggestions here.  this is for binaries that don't have an existing base.  should we make this an ELF config entry?